it/it_tenant_onboarding_test.go

```
@@ -0,0 +1,366 @@
+package it
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2/dsl/core"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)